kosat-core/src/commonMain/kotlin/org/kosat/SolverTypes.kt

```diff
@@ -61,6 +53,18 @@ value class Lit(val inner: Int) {
 
     /** Is the literal [Lit.UNDEF] or [Var.posLit]/[Var.negLit] of [Var.UNDEF] */
     val isUndef: Boolean get() = inner < 0
+
+    fun toDIMACS(): Int {
+        return if (isPos) variable.index + 1 else -(variable.index + 1)
+    }
+
+    companion object {
+        val UNDEF = Lit(-1)
```

Comment:
Always specify public field types.

Also, why it is called "undef"? Look like a sentinel value, which should not exist in the first place, but since we somehow need, maybe come up with a better naming?